packages/core/src/index.tsx

```diff
@@ -1,98 +1,112 @@
 
-    const [ghUserName, repoSlug] = repo.split("/")
+    const [ghUserName, repoSlug] = repo.split("/");
 
     return {
       ghToken,
       repo,
       ghUserName,
-      repoSlug
+      repoSlug,
     };
   })();
 
-  private PAGE_PATH_PLACEHOLDER = 'path'
+  private PAGE_PATH_PLACEHOLDER = "path";
 
-  private REVALIDATE = 60
+  private REVALIDATE = 60;
 
-  private allowUsers: string[] = [this.resolvedConfig.ghUserName].concat(this.config?.allowUsers || [])
+  private allowUsers: string[] = [this.resolvedConfig.ghUserName].concat(
+    this.config?.allowUsers || []
+  );
 
   private authHeaders = this.resolvedConfig.ghToken
     ? {
-      Authorization: `bearer ${this.resolvedConfig.ghToken}`,
-    }
+        Authorization: `bearer ${this.resolvedConfig.ghToken}`,
+      }
     : ({} as any);
 
-  private async request<T>(query: {
-    query: string,
-    variables: any
-  }) {
-    const res = await axios.post('https://api.github.com/graphql', query, {
-      headers: this.authHeaders
-    })
-    return res.data.data as T
+  private async request<T>(query: { query: string; variables: any }) {
+    const res = await axios.post("https://api.github.com/graphql", query, {
+      headers: this.authHeaders,
+    });
+    return res.data.data as T;
   }
 
-  constructor(public config: SairinConfig) {
-  }
+  constructor(public config: SairinConfig) {}
 
   getPostList = async () => {
+    const result = await this.request<GetPostQueryProps>(
+      GetPostsQuery({
+        owner: this.resolvedConfig.ghUserName,
+        repo: this.resolvedConfig.repoSlug,
+      })
+    );
 
-    const result = await this.request<GetPostQueryProps>(GetPostsQuery({
-      owner: this.resolvedConfig.ghUserName,
-      repo: this.resolvedConfig.repoSlug,
-    }))
-
-    const posts = result.repository.issues.nodes.filter(post => {
-      return this.allowUsers.indexOf(post.author.login) !== -1
-    }).map((post) => {
-      const { html, attributes } = this.processBody(post.body);
-      post.comments.nodes = post.comments.nodes.map(comment => {
-        const { html } = this.processBody(comment.body)
-        return {
-          ...comment,
-          html
-        }
+    const posts = result.repository.issues.nodes
+      .filter((post) => {
+        return this.allowUsers.indexOf(post.author.login) !== -1;
       })
+      .map((post) => {
+        const { html, attributes } = this.processBody(post.body);
+        post.comments.nodes = post.comments.nodes.map((comment) => {
+          const { html } = this.processBody(comment.body);
+          return {
+            ...comment,
+            html,
+          };
+        });
 
-      return {
-        ...post,
-        html,
-        attributes,
-      };
-    });
+        return {
+          ...post,
+          labels: post.labels.nodes.reduce(
+            (filtered, label) =>
+              label.name.startsWith("tag/")
```

Comment:
I think we can just use the origin tag. No need a `tag/` prefix

Reply:
then every post would have a `published` label, this might be redundant, adding a `tag/` prefix will be more robust to reflect the intention of using certain label as post tag and might leave more space for future leverage of github issue's label system

if you still want the simplicity, I could remove it, but do we want to keep the `published` tag?

Reply:
We can just filter out the `published` label here. What do you think?

Reply:
sure, let's just keep it simple~